applications/StructuralMechanicsApplication/custom_elements/nodal_concentrated_element.cpp

```diff
@@ -495,63 +461,23 @@ int NodalConcentratedElement::Check( const ProcessInfo& rCurrentProcessInfo )
 {    
     KRATOS_TRY
     
-    // Verify that the variables are correctly initialized
-
-    if ( VELOCITY.Key() == 0 )
-    {
-        KRATOS_ERROR << "VELOCITY has Key zero! (check if the application is correctly registered" << std::endl;
-    }
-
-    if ( DISPLACEMENT.Key() == 0 )
-    {
-        KRATOS_ERROR << "DISPLACEMENT has Key zero! (check if the application is correctly registered" << std::endl;
-    }
-
-    if ( ACCELERATION.Key() == 0 )
-    {
-        KRATOS_ERROR << "ACCELERATION has Key zero! (check if the application is correctly registered" << std::endl;
-    }
-
-    if ( NODAL_MASS.Key() == 0 )
-    {
-        KRATOS_ERROR << "NODAL_MASS has Key zero! (check if the application is correctly registered" << std::endl;
-    }
+    // Base class 
+    Element::Check(rCurrentProcessInfo);
     
-    if ( NODAL_STIFFNESS.Key() == 0 )
-    {
-        KRATOS_ERROR << "NODAL_STIFFNESS has Key zero! (check if the application is correctly registered" << std::endl;
-    }
-
-    if ( NODAL_DAMPING_RATIO.Key() == 0 )
-    {
-        KRATOS_ERROR << "NODAL_DAMPING_RATIO has Key zero! (check if the application is correctly registered" << std::endl;
-    }
-
-    if ( VOLUME_ACCELERATION.Key() == 0 )
-    {
-        KRATOS_ERROR << "VOLUME_ACCELERATION has Key zero! (check if the application is correctly registered" << std::endl;
-    }
-
-    for ( std::size_t i = 0; i < this->GetGeometry().size(); i++ )
-    {
-        if ( this->GetGeometry()[i].SolutionStepsDataHas( VOLUME_ACCELERATION ) == false )
-        {
-            KRATOS_ERROR << "Missing variable VOLUME_ACCELERATION on node " << this->GetGeometry()[i].Id() << std::endl;
-        }
-    }
-
-    // Verify that the dofs exist
-    for ( std::size_t i = 0; i < this->GetGeometry().size(); i++ )
-    {
-        if ( this->GetGeometry()[i].SolutionStepsDataHas( DISPLACEMENT ) == false )
-        {
-            KRATOS_ERROR << "Missing variable DISPLACEMENT on node " << this->GetGeometry()[i].Id() << std::endl;
-        }
-
-        if ( this->GetGeometry()[i].HasDofFor( DISPLACEMENT_X ) == false || this->GetGeometry()[i].HasDofFor( DISPLACEMENT_Y ) == false || this->GetGeometry()[i].HasDofFor( DISPLACEMENT_Z ) == false )
-        {
-            KRATOS_ERROR << "Missing one of the dofs for the variable DISPLACEMENT on node " << GetGeometry()[i].Id() << std::endl;
-        }
+    // Verify that the variables are correctly initialized
+    KRATOS_ERROR_IF(VELOCITY.Key() == 0) << "VELOCITY has Key zero! (check if the application is correctly registered" << std::endl;
```

Comment:
Can you use the functions from checks.h?

Reply:
Yep